sktime/markov/tools/estimation/sparse/mle/__init__.py

```diff
@@ -0,0 +1,84 @@
+import warnings
+import numpy as np
+import scipy
+from .. import _mle_sparse_bindings as _bindings
+
+
+def mle_trev(C, maxerr=1.0E-12, maxiter=int(1.0E6),
```

Comment:
why does this stuff go in an `__init__` file?

Reply:
doesn't have to, obviously. I just didn't feel like adding another source file. :man_shrugging:

Reply:
also it's mostly just setting up data for the c++ extension - if you feel strongly about it I can move it to another place though

Reply:
i don't feel strongly about it but neither neutral; i think it's not obvious that if someone were looking for this they'd go to `__init__`. i respect your choice :)